src/OrchardCore.Modules/OrchardCore.Layers/Controllers/AdminController.cs

```diff
@@ -128,7 +128,7 @@ public async Task<IActionResult> Index()
     }
 
     [HttpPost]
-    public async Task<IActionResult> Index(LayersIndexViewModel model)
+    public async Task<IActionResult> Index(LayersIndexViewModel _)
```

Comment:
why did you not just remove the parameter instead?

Reply:
but if the code isn't using it, its useless. We could rename it to `IndexPost` and add `[ActionName(nameof(Index))]` attribute.

Reply:
> but if the code isn't using it

I haven't checked. It's possible that client already pass a view model to the call and it may still be there because it was used at some point. Now if you remove the argument the behavior is different as passing something invalid would not fail without it. It's a breaking change to remove the argument.

Maybe it's the correct thing to do but that means checking our source for any usage of it. And then if we don't pass or need to pass anything to the call then document the potential breaking change. Hence using `_` is right now much easier.

Reply:
Filed an issue to track
https://github.com/OrchardCMS/OrchardCore/issues/16570

Reply:
We can create a PR after 2.0 since there is absolutely no harm it leaving it as it is.